packages/browser/src/client/orchestrator.ts

```diff
@@ -273,6 +273,21 @@ async function setIframeViewport(
   if (ui) {
     await ui.setIframeViewport(width, height)
   }
+  else if (getBrowserState().provider === 'playwright') {
```

Comment:
I couldn't figure out what's wrong with webdriverio, so I added scaling only for `playwright` and CI seems mostly happy.